app/services/catalog/update_order_item.rb

```diff
@@ -0,0 +1,33 @@
+module Catalog
+  class UpdateOrderItem
+    class OrderItemNotFound < StandardError; end
+
+    def initialize(topic)
+      @payload = topic.payload
+      @message = topic.message
+    end
+
+    def process
+      Rails.logger.info("Processing service order topic message: #{@message} with payload: #{@payload}")
+
+      Rails.logger.info("Searching for OrderItem with a task_ref: #{@payload["task_id"]}")
+      order_item = OrderItem.where(:topology_task_ref => @payload["task_id"]).first
+      raise OrderItemNotFound if order_item.nil?
+      Rails.logger.info("Found OrderItem: #{order_item.id}")
```

Comment:
@eclarizio 
I think we should set the context here similar to this
https://github.com/ManageIQ/catalog-api/blob/2586c59a45648208ad8ee6f9fcac6f3c7f989994/app/services/catalog/order_item_transition.rb#L13

It helps in log tracing